README.md

```diff
@@ -93,22 +94,44 @@ See [/examples/maildev](./examples/maildev) for a working example app using [Mai
-    - `context`: `object` - Object containing context for the message.
-  - Returns: `object` with following properties:
-    - `to`: `array` - Array containing recipient[s] email address (`string`).
-    - `from`: `string` - The email address the email is to be sent from.
-    - `html`: `string` - The HTML of the email with context injected.
-    - `subject`: `string` - The subject of the email with context injected.
-
-- `sendMail`: `function` - Calls `createMessage` to generate an message and uses [fastify-nodemailer](https://github.com/lependu/fastify-nodemailer) to send the generated email. 
+    - message: 'object'
+        - from: 'string' - The email address the email is to be sent from.
+        - to: 'array' - Comma separated list or an array of recipients email addresses (`string`) that will appear on the To: field
+        - cc: 'array' - Comma separated list or an array of recipients email addresses (`string`) that will appear on the Cc: field
+        - bcc: 'array' - Comma separated list or an array of recipients email addresses (`string`) that will appear on the Bcc: field
+        - replyTo : 'string' - An email address that will appear on the Reply-To: field
+        - subject: 'string' - The subject of the email with context injected.
+        - html: 'string' - The HTML version of the message as an Unicode string, with context injected.
+        - text : 'string' - The plaintext version of the message as an Unicode string, with context injected
+    - templatePath: 'string' - the relative path to the message's templates.
+  - opts: 'object' - Object containing options:
+      -  templatePath:  'string' - the relative path to the message's templates.
+      -  context: 'object' - Object containing context for the message (such as - variables that will be used in copy)
+
+  - Returns: 'object' with following properties:
+    - accepted : 'array' of email addresses accepted - eg. [ 'test@example.com' ]
+    - rejected : 'array' of email addresses rejected - eg. [],
+    - envelopeTime
+    - messageTime 
+    - messageSize 
+    - response 
+    - envelope 
+    - messageId
+
+- createMessage: 'function' - Generates a message object where the data provided is updated to use templates where available with context variables injected
   - Accepts the following arguments: 
-    - `recipients`: `array` - Array containing recipient[s] email address (`string`).
-    - `templates`: `string` - The relative path to the message's templates.
-    - `context`: `object` - Object containing context for the message.
+    - message: 'object'
+      - fields as above
+    - templatePath: 'string' - the relative path to the message's templates.
+    - context: 'object' - Object containing context for the message (such as - variables that will be used in copy)
+
+    For more details on this response see the Nodemail documentation [View nodemailer's docs here](https://nodemailer.com/smtp/)
+
+
+
```

Comment:
can clear some of these newlines out, just need one for the markdown to be useful